login-fire-form.html

```diff
@@ -287,9 +292,9 @@
     _signUp: function(e) {
         this._cleanMessages();
         this.$$("#repeatPasswordInput").invalid=false;
-        e = this.$.emailInput.validate()
-        p = this.$.passwordInput.validate()
-        pRepeat = this.password == this.passwordAgain
+        e = this.$$("#emailInput").validate()
```

Comment:
this.$.id breaks a lot for some reason that's why we switched to this.$$(). I guess we should switch to document.querySelector instead since it will be removed in v2. Thanks!